src/textbooks/textbook-card/TextbooksCard.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Collapsible,
+  Icon,
+  IconButtonWithTooltip,
+} from '@edx/paragon';
+import {
+  EditOutline as EditIcon,
+  RemoveRedEye as ViewIcon,
+  DeleteOutline as DeleteIcon,
+} from '@edx/paragon/icons';
+
+import messages from './messages';
+
+const TextbookCard = ({ title, chapters }) => {
+  const intl = useIntl();
+
+  return (
+    <div className="textbook-card" data-testid="textbook-card">
```

Comment:
Can we use here `Card` component from Paragon. The idea is to reuse Paragon components where it is possible.
I would consider to use the `Card` wrapper and the `Card.Header` with the `ActionRow` here.

For reference:

Similar component:
https://github.com/raccoongang/frontend-app-course-authoring/blob/4f2cf7308817488ea7da464f7d85a4cba895c586/src/course-unit/course-xblock/CourseXBlock.jsx#L39

Design for XBlock:
https://www.figma.com/file/YeKFwSpyLaJFDs3f3p3TSa/Studio-1%3A1-mock-ups?node-id=599%3A23626&mode=dev

Reply:
Fixed